### PR TITLE
feat: build feedbacks list page

### PR DESCRIPTION
**Pull request by mapra99**
product with feedback
image

product with no feedback
image

Steps to reproduce:
1. checkout to this branch
3. run `bin/dev/setup`
4. run services `docker compose up`
5. Go to http://localhost:3000 and login with these creds
```
email: rick@citadel.com
password: password
```

### CODE REVIEW

File: web/src/components/mobile-product-card/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export { default } from './mobile-product-card'
```
Comment (mapra99) [inline web/src/components/mobile-product-card/index.ts]:
```suggestion
export { default } from './mobile-product-card'

```

File: web/src/components/product-menu/index.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export { default } from './product-menu'
```
Comment (mapra99) [inline web/src/components/product-menu/index.ts]:
```suggestion
export { default } from './product-menu'

```

File: web/src/icons/menu-icon/menu-icon.tsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+export default function MenuIcon() {
+  return (
+    <svg viewBox="0 0 20 17" fill="none">
+      <rect id="Rectangle" width="20" height="3" fill="currentColor"/>
+      <rect id="Rectangle Copy" y="7" width="20" height="3" fill="currentColor"/>
+      <rect id="Rectangle Copy 2" y="14" width="20" height="3" fill="currentColor"/>
+    </svg>
+  )
+}
```
Comment (mapra99) [inline web/src/icons/menu-icon/menu-icon.tsx]:
```suggestion
}

```

**Review by ImMPrada — APPROVED** [review]
go go go!

**Pull request merged**